server/services/web.ts

```diff
@@ -28,6 +27,12 @@ const scriptSrc = [
   "cdn.zapier.com",
 ];
 
+// Allow to load assets from Vite
+if (!isProduction) {
+  scriptSrc.push("127.0.0.1:3001");
+  scriptSrc.push("localhost:3001");
+}
+
```

Comment:
Vite serves assets from another URL. We need to allow it for non-production environments.